package.json

```diff
@@ -31,7 +31,7 @@
     "swagger-editor": "^2.9.2",
     "swagger-tools": "^0.8.6",
     "async": "^1.2.1",
-    "swagger-test-templates": "0.0.0-alpha"
+    "swagger-test-templates": "^0.1.1"
```

Comment:
You need to rebase from master. This change was made in https://github.com/apigee-127/swagger-node/commit/353d6ec8c2f0ad868c74a79edc72a5321294e7bc